### PR TITLE
[Feature] Countrywise sales tax template

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
#5995

### CODE REVIEW

File: erpnext/setup/setup_wizard/data/country_wise_tax.json [hunk 1]
```diff
@@ -0,0 +1,1380 @@
...
+	"Uganda": {
+		"Uganda Tax": {
+			"account_name": "VAT",
+			"tax_rate": 18.00
+		}
+	},
+
+	"Ukraine": {
+		"Ukraine Tax": {
+			"account_name": "VAT",
```
Comment (strixaluco) [inline erpnext/setup/setup_wizard/data/country_wise_tax.json]: `"account_name": "ПДВ",`

**Review by strixaluco — COMMENTED** [review]
Ukraine Tax name fixed

File: erpnext/setup/setup_wizard/data/country_wise_tax.json (hunk 1 above)
Comment (PawanMeh) [inline erpnext/setup/setup_wizard/data/country_wise_tax.json]: Language translation should take care of this automatically
screen shot 2016-09-26 at 2 14 04 pm

Comment (strixaluco) [inline erpnext/setup/setup_wizard/data/country_wise_tax.json]: Great then)

**Comment by nabinhait** [comment]
merged from command line.

@rohitwaghchaure check the changes by me (https://github.com/frappe/erpnext/commit/6fb7b162e8b8698ef593599ae0dc16168dabc50c), removed duplicate functions.

**Pull request closed without merging**